postgres/init.sh

```diff
@@ -6,7 +6,7 @@ echo 'Setting up PostgreSQL database'
 # Replace with extracting username and password from config
 # Set password and utilize username from extracted var
 psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" <<-EOSQL
-    CREATE USER grafeas;
+    CREATE USER grafeas WITH PASSWORD 'changeme';
```

Comment:
Does this need to be the same password as in `docker-compose.yaml`?

Reply:
The `grafeas` user with the password `changeme` is used in the server configuration https://github.com/grafeas/grafeas-pgsql/blob/master/server/config.yaml#L37. 
The `docker-compose.yaml` has a postgres user/pswd to start the postgres instance.
We could make it all the same if this leads to confusion.

Reply:
Ah, gotcha, good point. Let's leave this as is for now.